src/modules/matrix/MatrixAllView.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="background">
+    <ToolBar/>
+    <v-container fluid>
+      <v-row align="center" justify="center">
+        <v-col md="8" sm="10" xs="14">
+          <v-card>
+            <v-toolbar
+              color="primary" 
+              class="white--text"
+              height="60px"
+              width="100%" 
+              flat
+            >
+              <v-toolbar-title v-text="$t('views.matrix.all.title')" />
+              <v-spacer/>
+              <v-toolbar-items>
+                <v-flex mt-5>
+                  <v-checkbox
+                    dark
+                    input-value="true"
+                    value
+                    hide-details
+                    label="Status"
+                  />
+                </v-flex>
+              </v-toolbar-items>
+            </v-toolbar>
+            <v-simple-table>
+              <template>
+                <thead>
+                  <tr>
+                    <th/>
+                    <th class="text-center" v-text="$t('views.matrix.all.headers.matrix')" />
+                    <th class="text-center" v-text="'Status'" />
+                    <th class="text-center" ><v-icon v-text="'mdi-pencil'" /></th>
+                    <th class="text-center" ><v-icon v-text="'mdi-check'" /></th>
+                    <th class="text-center" ><v-icon v-text="'mdi-close'" /></th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr
+                    v-for="matrix in matrices"
+                    :key="matrix.id"
+                  >
+                    <td class="text-center" ><v-icon v-text="'mdi-pig'" /></td>
+                    <td class="text-center" v-text="matrix.matrix" />
+                    <td class="text-center" v-text="$t(`fields.${matrix.status.toLowerCase()}`)" />
+                    <td class="text-center" ><v-btn x-small color="secondary" class="white--text" :to="''" v-text="$t('buttons.update')" /></td>
```

Comment:
Precisa desses `:to=""'''"` vazio?

Reply:
A ideia é ser preenchido (ou removido) quando as requests forem montadas